patches/minecraft/net/minecraft/network/NettyPacketEncoder.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/network/NettyPacketEncoder.java
++++ b/net/minecraft/network/NettyPacketEncoder.java
+@@ -37,7 +37,7 @@
+             try {
+                p_encode_2_.func_148840_b(packetbuffer);
+             } catch (Throwable throwable) {
+-               field_150798_a.error(throwable);
++               field_150798_a.error(String.format("Critical failure encoding packet: %s", packetbuffer.toString()), throwable); // FORGE: Get Minecraft to spit out more information about the Throwable we got.
+                if (p_encode_2_.func_211402_a()) {
```

Comment:
Using packetbuffer.toString() is just asking for trouble. Probably better to be throwable.getMessage() or just "Error encoding packet" and let the stack trace deal with it.

Reply:
Changed to just "Error encoding packet" to let the `Throwable` do its thing.